cmd/relayproxy/config/config.go

```diff
@@ -112,6 +119,14 @@ func New(flagSet *pflag.FlagSet, log *zap.Logger, version string) (*Config, erro
 	if errUnmarshal != nil {
 		return nil, errUnmarshal
 	}
+
+	if proxyConf.Debug && proxyConf.LogLevel == "" {
+		log.Warn(
+			"Option Debug that you are using in your configuration file is deprecated" +
+				"and will be removed in future versions." +
+				"Please use logLevel: debug to continue to run the relay-proxy with debug logs.")
+	}
```

Comment:
`proxyConf.LogLevel == ""` will never be true because you have a default value.


```suggestion
	if proxyConf.Debug {
		log.Warn(
			"Option Debug that you are using in your configuration file is deprecated " +
				"and will be removed in future versions." +
				"Please use logLevel: debug to continue to run the relay-proxy with debug logs.")
	}
```